### PR TITLE
Update kvproto to take the change of defining new TSO service

### PR DESCRIPTION
**Pull request by binshi-bing**
Issue Number: ref #5836

Signed-off-by: binshi-bing <binshi.bing@gmail.com>

### What problem does this PR solve?

Issue Number: Close #xxx

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- add gPRCs for the independent TSO service

### Release note

```release-note
None
```

### CODE REVIEW

**Review by JmPotato — COMMENTED** [review]
Usually, we don't file and merge a PR that only updates kvproto to a non-final version. This should be proposed with the corresponding code changes to ensure everything is OK before merging the breaking change into the master.

**Comment by binshi-bing** [comment]
> 

Make sense. I'll hold the PR and continue making the change.

**Review by lhy1024 — COMMENTED** [review]
only update kvproto？

**Comment by lhy1024** [comment]
> Usually, we don't file and merge a PR that only updates kvproto to a non-final version. This should be proposed with the corresponding code changes to ensure everything is OK before merging the breaking change into the master.

I agree.

**Comment by binshi-bing** [comment]
reopen it as I closed it by accident. Will handle feedback later.

File: client/go.mod [hunk 1]
```diff
@@ -37,3 +37,5 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
```
Comment (lhy1024) [inline client/go.mod:40]: We can wait to update until kvproto merged

**Comment by binshi-bing** [comment]
replaced by https://github.com/pingcap/kvproto/pull/1053

**Comment by binshi-bing** [comment]
replaced by https://github.com/pingcap/kvproto/pull/1053

**Pull request closed without merging**